qml/Ping360Visualizer.qml

```diff
@@ -22,6 +22,13 @@ Item {
         property var ping: DeviceManager.primarySensor
         target: ping
 
+        /** Ping360 does not handle auto range/scale
+         *  Any change in scale is a result of user input
+         */
+        onLength_mmChanged: {
```

Comment:
I think this is better than the current behavior. In the future, it will be even better if we do not clear the the existing data that is remains inside the plot limits.